lib/Doctrine/ORM/Configuration.php

```diff
@@ -936,4 +936,24 @@ public function setDefaultQueryHint($name, $value)
     {
         $this->_attributes['defaultQueryHints'][$name] = $value;
     }
+
+    /**
+     * Gets a list of entity class names to be ignored by the SchemaTool
+     *
+     * @return string[]
+     */
+    public function getSchemaIgnoreClasses() : array
```

Comment:
I would like to make the naming a bit more precise, otherwise everything else is great. What do you think of `get/setEntitiesIgnoredDuringSchemaGeneration`. I first thought it should be classes instead of entities, but embeddables and other non-entities are skipped already, so its really about skipping entities here.

Reply:
OK, I'm happy with that. Apologies for the delay, just committed the changes here so fingers crossed.